lms/values.py

```diff
@@ -16,3 +16,34 @@ class LTIUser(NamedTuple):
 
     roles: str
     """The user's LTI roles."""
+
+
+class HUser(NamedTuple):
+    """
+    An 'h' user.
+
+    The lms app generates h user accounts that correspond to LMS users who
+    launch activities using the app. These user accounts have auto-generated
+    usernames.
+    """
+
+    authority: str
+    """
+    The authority which the user belongs to.
+
+    This should always match ``settings["h_authority"]``
+    """
+
+    display_name: str
+    """The display name for the user, generated from the LMS user's display name."""
+
+    username: str
+    """
+    Generated username for h user.
+
+    This is derived from the LMS userid.
+    """
+
```

Comment:
I haven't added the `provider` and `provider_unique_id` fields yet because they are not used in this PR.